apps/testingaccessibility/src/pages/api/auth/[...nextauth].ts

```diff
@@ -10,6 +10,53 @@ import {getSdk} from '../../../lib/prisma-api'
 import {defineRulesForPurchases} from '../../../server/ability'
 import mjml2html from 'mjml'
 
+export type MagicLinkEmailType =
+  | 'login'
+  | 'signup'
+  | 'reset'
+  | 'purchase'
+  | 'upgrade'
+
+export const sendVerificationRequest = async (params: {
```

Comment:
this was previously inline below and couldn't be accessed. This meant our custom template wasn't being used! It also made it impossible to use alternative templates/subjects for different email types (new purchase, upgrade, login)